ui/search.html

```diff
@@ -34,7 +34,7 @@
                             <div class="oae-clip-fold-outer"></div>
                         </div>
                         <div class="clearfix oae-clip-content">
-                            <form id="search-form" class="form-search form-horizontal" role="search" aria-label="__MSG__SEARCH__">
+                            <form action="." id="search-form" class="form-search form-horizontal" role="search" aria-label="__MSG__SEARCH__">
```

Comment:
@sathomas : Can we do the same for the other search forms in the application (landing page, top navigation, etc.) ?